flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/utils/CountAggFunction.scala

```diff
@@ -42,7 +42,7 @@ class CountAggFunction extends AggregateFunction[JLong, CountAccumulator] {
     acc.f0 += 1L
   }
 
-  def retract(acc: CountAccumulator, @DataTypeHint("INT") value: Any): Unit = {
+  def retract(acc: CountAccumulator, value: Any): Unit = {
```

Comment:
Is this change mandatory or is the annotation just no longer necessary?

Reply:
It was never necessary